src/import-export/import-main-matcher.c

```diff
@@ -481,10 +477,6 @@ gnc_gen_trans_init_view (GNCImportMainMatcher *info,
              "cell-background", DOWNLOADED_COL_COLOR,
              NULL);
     renderer = gtk_cell_renderer_text_new();
-    g_object_set(G_OBJECT(renderer),
-                 "foreground", "black",
-                 "foreground-set", TRUE,
-                 NULL);
```

Comment:
Hmm, as with the dense cal, I think you should set a text color at the same place you set a background color. Since we define default background colors based on the intervention required, we should set accompanying foreground colors IMO. Otherwise it will depend on the theme whether the text rendered is readable on our chose backgrounds. Moreover I even think we should define separate foreground colors for each of our 3 classes.